### PR TITLE
Replace heavyweight aws-java-sdk with fine-grained plugin

### PR DESCRIPTION
**Pull request by Vlatombe**
Downstream PR of
* https://github.com/jenkinsci/aws-credentials-plugin/pull/93

- [X] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [X] Ensure that the pull request title represents the desired changelog entry
- [X] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -106,7 +105,7 @@ THE SOFTWARE.
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>aws-credentials</artifactId>
-            <version>1.30</version>
+            <version>1.31-rc150.2d46aabfddaa</version><!-- https://github.com/jenkinsci/aws-credentials-plugin/pull/93 -->
```
Comment (Vlatombe) [inline pom.xml:108]:
```suggestion
            <version>1.32</version>
```

**Review by res0nance — APPROVED** [review]

**Comment by Vlatombe** [comment]
@res0nance would it be possible for you to merge/release this soon?

**Comment by res0nance** [comment]
> @res0nance would it be possible for you to merge/release this soon?

Sure, I assume your internal testing has not revealed any issues?

**Comment by Vlatombe** [comment]
No.

**Pull request merged**